packages/ocr_extraction/requirements.txt

```diff
@@ -0,0 +1,9 @@
+fastapi
+uvicorn
+pydantic
+python-multipart
+docling==2.25.2
+augraphy
+rapidocr-onnxruntime
+openai
+git+https://github.com/DARPA-ASKEM/TexTeller.git
```

Comment:
Could this be in terarium repo?

Reply:
Probably but I don't know pip well enough to wrangle that because you have a situation of having a repo inside a repo. Having said that I don't think it is a good idea to subsume library dependencies.